### PR TITLE
KTOR-6963 Fix darwin maxFrameSize option

### PR DESCRIPTION
**Pull request by e5l**
Fix [KTOR-6963](https://youtrack.jetbrains.com/issue/KTOR-6963) Darwin: The `maxFrameSize` option has no effect

### CODE REVIEW

File: ktor-client/ktor-client-core/common/src/io/ktor/client/plugins/websocket/WebSockets.kt [hunk 1]
```diff
@@ -185,6 +185,11 @@ public class WebSockets internal constructor(
                 }
 
                 LOGGER.trace("Receive websocket session from ${context.request.url}: $session")
+
+                if (plugin.maxFrameSize != Long.MAX_VALUE) {
```
Comment (marychatte) [inline ktor-client/ktor-client-core/common/src/io/ktor/client/plugins/websocket/WebSockets.kt:189]: Why compare with `Long.MAX_VALUE` when the default value is `Int.MAX_VALUE.toLong()`? https://github.com/ktorio/ktor/blob/3cd270690ca092fd3c01f82e2655821ec587009f/ktor-client/ktor-client-core/common/src/io/ktor/client/plugins/websocket/WebSockets.kt#L117

**Comment by marychatte** [comment]
Tests are falling for OkHttp engine :(

File: ktor-client/ktor-client-core/common/src/io/ktor/client/plugins/websocket/WebSockets.kt (hunk 1 above)
Comment (e5l) [reply ktor-client/ktor-client-core/common/src/io/ktor/client/plugins/websocket/WebSockets.kt:189]: Good catch, will fix
```suggestion
                if (plugin.maxFrameSize != Int.MAX_VALUE.toLong()) {
```

**Comment by e5l** [comment]
@marychatte, fixed with the fix of default check

**Review by marychatte — APPROVED** [review]
lgtm

**Pull request merged**